src/fprime/util/string_util.py

```diff
@@ -92,7 +92,7 @@ def convert_ignore_int(match_obj):
         result = formatted_str.format(*values)
         result = result.replace("%%", "%")
         return result
-    except ValueError as e:
+    except ValueError:
```

Comment:
It seems there is this exact pattern that's used [a couple lines below](https://github.com/fprime-community/fprime-tools/blob/53ccc63bc241c3a902e7d5c114fcf85d5ae178cf/src/fprime/util/string_util.py#L110), we should probably keep it consistent. Would you mind making that change as well?

Reply:
Oh wait I may have spoken too fast, we're raising it up in that second case.
https://github.com/fprime-community/fprime-tools/blob/6d258007391e9a4091b29b0af3ba6ef13c798f5b/src/fprime/util/string_util.py#L116

Reply:
Annnd we're actually using the exception...
https://github.com/fprime-community/fprime-tools/blob/6d258007391e9a4091b29b0af3ba6ef13c798f5b/src/fprime/util/string_util.py#L114
Read tgat way too fast lol, my apologies. This looks good to me.